native/cocos/renderer/GFXDeviceManager.h

```diff
@@ -110,8 +110,8 @@ class CC_DLL DeviceManager final {
         return nullptr;
     }
 
-    static constexpr bool isDetachDeviceThread() {
-        return DETACH_DEVICE_THREAD;
+    static bool isDetachDeviceThread() {
+        return DETACH_DEVICE_THREAD && Device::isSupportDetachDeviceThread;
```

Comment:
I suggest changing to
```c++
static constexpr bool isDetachDeviceThread() {
#if XR_OEM_PICO
    constexpr bool isSupportDetachDeviceThread{false};
#else
    constexpr bool isSupportDetachDeviceThread{true};
#endif
    return DETACH_DEVICE_THREAD && isSupportDetachDeviceThread;
}
```
And remove isSupportDetachDeviceThread from `Device`.
The reason is that the behavior of `Device::isSupportDetachDeviceThread` depends on the side-effects of user's setting. It is hard to maintain.

Reply:
isDetachDeviceThread() will be moved to Device in 3.6.2